pkg/smartcontract/manifest/standard/nep22.go

```diff
@@ -0,0 +1,25 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+// Nep22 is a NEP-22 Standard.
```

Comment:
Let's extend the doc, there's too many standards. `Nep22 is a NEP-22 Standard describing smart contract update functionality.` Ditto for NEP31.